subworkflows/nf-core/bam_sort_samtools/main.nf

```diff
@@ -2,23 +2,20 @@
 // Sort, index BAM file and run samtools stats, flagstat and idxstats
 //
 
-params.sort_options  = [:]
-params.index_options = [:]
-params.stats_options = [:]
-
-include { SAMTOOLS_SORT      } from '../../../modules/nf-core/samtools/sort/main'  addParams( options: params.sort_options  )
-include { SAMTOOLS_INDEX     } from '../../../modules/nf-core/samtools/index/main' addParams( options: params.index_options )
-include { BAM_STATS_SAMTOOLS } from '../bam_stats_samtools/main'           addParams( options: params.stats_options )
+include { SAMTOOLS_SORT      } from '../../../modules/nf-core/samtools/sort/main'
+include { SAMTOOLS_INDEX     } from '../../../modules/nf-core/samtools/index/main'
+include { BAM_STATS_SAMTOOLS } from '../bam_stats_samtools/main'
 
 workflow BAM_SORT_SAMTOOLS {
     take:
-    ch_bam // channel: [ val(meta), [ bam ] ]
+    bam   // channel: [ val(meta), [ bam ] ]
+    fasta // channel: [ fasta ]
 
     main:
 
     ch_versions = Channel.empty()
 
-    SAMTOOLS_SORT ( ch_bam )
+    SAMTOOLS_SORT ( bam )
```

Comment:
why not keep `ch_bam`, if you have `ch_bam_bai`?

Reply:
Yeah, I debated about that to myself before I made this change. You are probably right. I'll update.

Reply:
I though the consensus was to prefix all channels with `ch_`?  I could swear I read that somewhere...

Reply:
It was. But in some instances we may be passing a `file` object instead of a channel like `fasta` where this wouldn't apply. In these scenarios, `ch_` may not necessarily apply. Fine to have it in for all things files/channels and not for booleans and other types of objects.